translations/en.json

```diff
@@ -1543,7 +1568,8 @@
     "word-count": "Word count",
     "word-n": "Word #{{number}}",
     "word-n-description": "In order to check if you have backed up your seed phrase correctly, enter the word #{{number}} above.",
-    "words-n": {
+    "words-n":
```

Comment:
seems like you have an auto formatter that is adding some noise here? Maybe it's best to address so it's not doing the same in other pr's reversing these changes?

Reply:
I have been fighting to fix this. My Emacs go for 2 space tabs (the repo uses 4). And `lint-fix` doesn't fix it, so I had to use Sublime's formatter.

I'll check how to fix this.